.travis.yml

```diff
@@ -17,6 +17,7 @@ cache:
     - $HOME/.ghc
     - $HOME/.cabal
     - $TRAVIS_BUILD_DIR/.cache
+    - $TRAVIS_BUILD_DIR/node_modules
```

Comment:
If we're going to do this, then after `npm install`, we need to always run `npm prune`.